core/templates/dev/head/pages/admin/admin.html

```diff
@@ -69,6 +69,53 @@
+    <script src="/templates/dev/head/pages/collection_editor/collection-editor-navbar/collection-editor-navbar.directive.js"></script>
+
+    <script src="/templates/dev/head/pages/collection_editor/CollectionEditorStateService.js"></script>
+    
+    <!--CollectionEditorTabDirective.js-->
+    <script src="/templates/dev/head/pages/collection_editor/editor_tab/collection-editor-tab.module.js"></script>
+    <script src="/templates/dev/head/pages/collection_editor/editor_tab/collection-editor-tab.directive.js"></script>
+
+    <script src="/templates/dev/head/pages/collection_editor/editor_tab/CollectionLinearizerService.js"></script>
+    <!--CollectionNodeCreatorDirective.js-->
+    <script src="/templates/dev/head/pages/collection_editor/editor_tab/collection-node-creator/collection-node-creator.module.js"></script>
+    <script src="/templates/dev/head/pages/collection_editor/editor_tab/collection-node-creator/collection-node-creator.directive.js"></script>
+
+    <!--CollectionNodeEditorDirective.js-->
+    <script src="/templates/dev/head/pages/collection_editor/editor_tab/collection-node-editor/collection-node-editor.module.js"></script>
+    <script src="/templates/dev/head/pages/collection_editor/editor_tab/collection-node-editor/collection-node-editor.directive.js"></script>
+
+    <!--CollectionHistoryTabDirective.js-->
+    <script src="/templates/dev/head/pages/collection_editor/history_tab/collection-history-tab.module.js"></script>
+    <script src="/templates/dev/head/pages/collection_editor/history_tab/collection-history-tab.directive.js"></script>
+
+    <!--CollectionDetailsEditorDirecive.js-->
+    <script src="/templates/dev/head/pages/collection_editor/settings_tab/collection-details-editor/collection-details-editor.module.js"></script>
+    <script src="/templates/dev/head/pages/collection_editor/settings_tab/collection-details-editor/collection-details-editor.directive.js"></script>
+
+    <!--CollectionPermissionsCardDirective.js-->
+    <script src="/templates/dev/head/pages/collection_editor/settings_tab/collection-permissions-card/collection-permissions-card.module.js"></script>
+    <script src="/templates/dev/head/pages/collection_editor/settings_tab/collection-permissions-card/collection-permissions-card.directive.js"></script>
+
+    <!--CollectionSettingsTabDirective.js-->
+    <script src="/templates/dev/head/pages/collection_editor/settings_tab/collection-settings-tab.module.js"></script>
+    <script src="/templates/dev/head/pages/collection_editor/settings_tab/collection-settings-tab.directive.js"></script>
+
+    <!--CollectionStatisticsTabDirective.js-->
+    <script src="/templates/dev/head/pages/collection_editor/statistics_tab/collection-statistics-tab.module.js"></script>
+    <script src="/templates/dev/head/pages/collection_editor/statistics_tab/collection-statistics-tab.directive.js"></script>
```

Comment:
Group the script tags of the same page as one rather than subgrouping them further.

Reply:
Alright, Should I remove the comments or keep them?